package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "big.js": "^6.0.1",
+    "prop-types": "^15.7.2",
```

Comment:
# ADD react-redux as a dependency.

- [ ] Make sure `redux` is added to this `package.json` file as a dependency [screenshot from Pathwright - redux and react-redux](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone1/AddReduxAndReact-Redux.png).